Smartsheet.Net.Standard/Interfaces/ISmartsheetHttpClient.cs

```diff
@@ -27,6 +27,11 @@ public interface ISmartsheetHttpClient
 		Task<Workspace> GetWorkspaceById(long? workspaceId, string accessToken = null, bool loadAll = false);
 		Task<IEnumerable<Workspace>> ListWorkspaces(string accessToken = null);
 		
+		//  Shares
+		Task<IEnumerable<Share>> ShareWorkspace(long? workspaceId, IEnumerable<Share> shares, bool sendEmail = false, string accessToken = null);
+		Task<IEnumerable<Share>> ShareReport(long? reportId, IEnumerable<Share> shares, bool sendEmail = false, string accessToken = null);
+		Task<IEnumerable<Share>> ShareSheet(long? sheetId, IEnumerable<Share> shares, bool sendEmail = false, string accessToken = null);
+		Task<IEnumerable<Share>> ShareDashboard(long? dashboardId, IEnumerable<Share> shares, bool sendEmail = false, string accessToken = null);
```

Comment:
Though it's unfortunate, they've gone back and forth between 'Sight' and 'Dashboard' as the official name a few times, and despite the current version being called 'Dashboard' the API refers to the objects as 'Sights' and all fo the endpoints are under the Sight moniker. Therefore, to have better alignment with the API, this should be `ShareSight()` and use terms like `sightId` instead of `dashboardId` all throughout.

Reply:
Updated